packages/aws-cdk-lib/cloud-assembly-schema/lib/cloud-assembly/metadata-schema.ts

```diff
@@ -231,10 +255,36 @@ export type LogicalIdMetadataEntry = string;
  */
 export type StackTagsMetadataEntry = Tag[];
 
+/**
+ * @see ArtifactMetadataEntryType.ACKNOWLEDGE
+ */
+export interface AcknowledgementMetadataEntry {
```

Comment:
Why do acknowledgements need to make it into the Cloud Assembly? Can't we choose to not emit acknowledged warnings?